src/Symfony/Component/EventDispatcher/EventListenerInterface.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\EventDispatcher;
+
+/**
+ * Marker interface for event listeners.
+ *
+ * @author Jérémy Derussé <jeremy@derusse.com>
```

Comment:
what about adding `@method __invoke()` + some words about it?

Reply:
done (in interface body)